### ISSUE TITLE
GT question

### ISSUE THREAD

**Issue by cedricgsh** [ZhangGongjie/MapVR#4]
@ZhangGongjie Hello，thank for your amazing job. And I have a question about GT. The gt rasterized map should be float number after you convert points to map. However, you apply dice loss to optimize, do you convert gt map, a float number mask, to 0-1 mask for dice loss computing ?

**Comment by ZhangGongjie** [comment]
no, we didn't do any thresholding. Dice loss can apply to any value between 0 and 1 when used for segmentation.
